tests/test_tracing.py

```diff
@@ -271,14 +285,19 @@ def extractor_raiser(event, context):
             extractor=extractor_raiser,
         )
         self.assertEqual(ctx_source, "xray")
-        self.assertDictEqual(
-            ctx,
-            {
-                "trace-id": fake_xray_header_value_root_decimal,
-                "parent-id": fake_xray_header_value_parent_decimal,
-                "sampling-priority": "2",
-            },
+        self.assertEqual(
+            ctx.trace_id,
+            fake_xray_header_value_root_decimal
         )
+        self.assertEqual(
+            ctx.span_id,
+            fake_xray_header_value_parent_decimal
+        ) 
+        self.assertEqual(
+            ctx.sampling_priority,
+            '2'
+        ) 
```

Comment:
Go ahead and put this all on one line. We dont need this file longer than it already is.